content/source/docs/enterprise/api/workspaces.html.md

```diff
@@ -12,40 +12,98 @@ Workspaces represent running infrastructure managed by Terraform.
+By supplying the necessary attributes under a `vcs-repository` object, you can create a Workspace that is configured against a VCS Repository.
 
-#### Parameters
+By supplying the necessary attribute, `migration-environment`, you can create a workspace which is migrated from a legacy environment. When you do this, the following will happen:
+
+* Environment and Terraform variables will be copied to the workspace.
+* Teams which are associated with the legacy environment will be created in the destination workspace's organization, if teams with those names don't already exist.
+* Members of those teams will be added as members of the corresponding teams in the destination workspace's organization.
+* Each team will be given the same access level on the workspace as it had on the legacy environment.
+* The latest state of the legacy environment will be copied over into the workspace and set as the workspace's current state.
+* VCS repo ingress settings (like branch and working directory) will be copied over into the workspace.
+
+| Key path                                      | Type    | Default    | Description                                                                                                                                                                                                                             |
+| --------------------------------------------- | ------- | ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `data.type`                                   | string  |            | Must be `"workspaces"`.                                                                                                                                                                                                                 |
+| `data.attributes.name`                        | string  |            | Specifies the name of the workspace, which can only include letters, numbers, `-`, and `_`. This will be used as an identifier and must be unique in the organization.                                                                  |
+| `data.attributes.auto-apply`                  | boolean | false      | Whether to automatically apply changes when a Terraform plan is successful.                                                                                                                                                             |
+| `data.attributes.terraform-version`           | string  |            | The version of Terraform to use for this workspace.                                                                                                                                                                                     |
```

Comment:
Default for this could use some elaboration... Also, I wonder how we determine the legal values? It's a drop-down in the ui, right?